community-tutorials/dockerized-csgo-gameserver/01-en.md

```diff
@@ -0,0 +1,337 @@
+Here let ```user@example.com``` be your ssh user and host then simply run:
+```bash
+docker -H "ssh://user@example.com" build . -t csgo
+```
+to build the container and run:
+```bash
+docker -H "ssh://user@example.com" run -d --restart=unless-stopped --name csgo \
+    -p 27015:27015/udp -p 27015:27015 -it -v csgo_data:/data csgo
+```
+to start the container with default ports. If you plan on running multiple containers or prefer different ports, you need to adjust the port mappings accordingly with the ```-p``` argument.
+
+If your created all files directly on your server, simply omit the ```-H "ssh://user@example.com"``` argument.
+
+Now the container should be started and will begin downloading the CS:GO dedicated server.
+This can take some minutes, since there are about 20GB of server files to be downloaded.
+
+You can see the logs by running:
+```bash
+docker -H "ssh://user@example.com" logs -f csgo
+```
+
+You can also easily start, restart and stop the container by running:
+```bash
+docker -H "ssh://user@example.com" start csgo
+docker -H "ssh://user@example.com" restart csgo
+docker -H "ssh://user@example.com" stop csgo
+```
+
+If you want to get shell access inside the container to edit config files, you can do that by running: 
+```bash
+docker -H "ssh://user@example.com" exec -it csgo bash
+```
+Alternatively you sometimes might need access to the command line interface for the CS:GO server if you don't want to use rcon.
+This can be done by attaching to the container by running:
+```bash
+# this is essentially the same as opening your console in the csgo client
+docker -H "ssh://user@example.com" attach csgo
+```
+
+Another useful thing are backups, which are not really needed in this example, since most of the servers state is configured in the config files, but still can be useful.
+To copy files from the container to your computer, you can run:
+```bash
+docker -H "ssh://user@example.com" cp csgo:/data/csgo_ds/csgo/cfg/<files to copy> .
+``` 
+Alternatively you can also mount a filesystem directly into the container:
+```bash
+# this mounts csgo_data to /data and /backup on the server to /backup inside the container
+docker run -d --rm -it -v csgo_data:/data -v /backup:/backup ubuntu
+```
+then simply copy all the files you want to backup from ```/data``` to ```/backup``` inside the container.
+This requires that you previously created a directory named ```/backup``` on your server.
+This is where the copied files from the container will end up.
+# Conclusion
+Now you should have a running CS:GO dedicated server that keeps itself up to date.
+All data is stored in the ```csgo_data``` docker volume, which persists the data even of you remove the container.
+
+You can now connect to your server at ```steam://connect/<example.com>/<sv_password>``` or via the community server browser in game.
+
+All steps are essentially the same for other source games from Valve, such as Garrys Mod or CSS.
+
```

Comment:
It is difficult to distinguish which commands I have to execute locally and which I have to execute on the server.
Please divide it better (locally or on the server).
Above all, it is primarily important how I set it up on the Netcup product. 
The local variant is rather secondary.